catalog-rest-service/src/main/java/org/openmetadata/catalog/security/OAuthProxyAuthFilter.java

```diff
@@ -0,0 +1,78 @@
+package org.openmetadata.catalog.security;
+
+import com.google.common.base.Strings;
+import org.openmetadata.catalog.security.auth.CatalogSecurityContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.UriInfo;
+import java.lang.invoke.MethodHandles;
+
+public class OAuthProxyAuthFilter implements ContainerRequestFilter  {
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
There is an @Slf4j annotation from lombok that we can use to avoid this line altogether.

The default variable is called `log` -- this can be customized with lombok.config